controllers/profiles/reconciler_dev.go

```diff
@@ -267,7 +273,7 @@ func getDeploymentFailureReasonOrDefaultReason(deployment *appsv1.Deployment) st
 }
 
 // mountDevConfigMapsMutateVisitor mounts the required configMaps in the Workflow Dev Deployment
-func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVisitor {
+func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap, support *stateSupport, workflow *operatorapi.KogitoServerlessWorkflow) mutateVisitor {
```

Comment:
Rather, please pass an array of ConfigMaps instead. The mutators don't have access to the client connector on purpose. Their sole role is to guarantee the object structure, not fetch the API.

Reply:
Ok